samples/dnn/text_detection.cpp

```diff
@@ -159,3 +203,59 @@ void decode(const Mat& scores, const Mat& geometry, float scoreThresh,
         }
     }
 }
+
+void four_points_transform(const Mat& frame, const Size2f& box_size, Point2f vertices[4], Mat& result) {
```

Comment:
What is the source of this implementation?

Reply:
I took source from here https://www.pyimagesearch.com/2014/08/25/4-point-opencv-getperspective-transform-example/